src/Horse.Jhonson.pas

```diff
@@ -34,6 +34,10 @@ procedure Jhonson(Req: THorseRequest; Res: THorseResponse; Next: TProc);
   begin
     LWebResponse.Content := TJSONValue(LContent).ToJSON;
     LWebResponse.ContentType := 'application/json';
+  end
+  else if not Assigned(LContent) then
```

Comment:
I believe that this verification of `Assigned` is not necessary, since the first `if` already does this verification!

Reply:
Bom dia, não me atentei... Obrigado